circle.yml

```diff
@@ -1,26 +0,0 @@
-machine:
-  environment:
-    KITCHEN_LOCAL_YAML: .kitchen.docker.yml
-  services:
-    - docker
-
-dependencies:
-  override:
-    - curl -L https://www.chef.io/chef/install.sh | sudo bash -s -- -P chefdk -v 0.12.0
-    - chef gem install kitchen-docker -v '~> 2.3.0'
-  cache_directories:
-    - "~/.chefdk"
-
-test:
-  pre:
-    - eval "$(/opt/chefdk/bin/chef shell-init bash)"
-  override:
-    - chef --version
-    - chef gem query -d chefspec rubocop foodcritic rake kitchen-docker
-    - chef exec rake circle:
-        parallel: true
```

Comment:
Do we need to add `parallel` and `timeout` to the new circleci file ?

Reply:
I couldn't find the way to add the directives to the job in `v2`. I'll take another look.

Reply:
OK, we should just double check what they did in V1 and that we no longer need them with V2.

Reply:
setting parallelism would not help in circleci v2. It didn't really accomplish much in v1 anyways.